shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/target/SurveyTargetType.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.blisskmmic.data.network.target
+
+import co.nimblehq.blisskmmic.data.network.helpers.TargetType
+import io.ktor.http.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+sealed class SurveyTargetType<T>: TargetType<T>
```

Comment:
We can improve the generic type
```suggestion
sealed class SurveyTargetType<BodyType>: TargetType<BodyType>
```